### ISSUE TITLE
Output of Gan Images is just Black

### ISSUE THREAD

**Issue by nareshmungpara** [deepak112/Keras-SRGAN#24]
![generated_image_5](https://user-images.githubusercontent.com/34807837/66919427-ec068680-f03e-11e9-9c0f-ecb9bf653d5b.png)

I am getting output of test like above shown image.

When I am training at that time as well it is generating similar images in output folder.
I had train for 100 text images with 1000 epochs. 

Please suggest some way to resolve it.

**Comment by Sylv-Lej** [comment]
Try de-normalize your image before plotting it !
